src/constants.js

```diff
@@ -29,4 +29,39 @@ module.exports = {
     {name: "XOle", mirror: false, shortBurst: true},
     {name: "XSlide", mirror: false, shortBurst: true},
   ],
+  RANDOM_EFFECT_KEY: 'rand',
+  BACKGROUND_EFFECTS: [
```

Comment:
Is it possible / realistic that this list gets out of sync and contains a background which doesn't exist yet / no longer exists? Where's the authoritative source of implemented backgrounds? If so, is it possible to fail gracefully if we randomly pick one which doesn't exist?

Reply:
That's a great point.  `Effects.js` contains the actual implemented effects.  I'll union this list of 'approved' effects for random selection with the set of implemented effects before we pick one.